examples/truffle/package.json

```diff
@@ -23,7 +23,7 @@
   "homepage": "https://github.com/gnosis/ethcontract-rs",
   "devDependencies": {
     "@gnosis.pm/util-contracts": "^2.0.6",
-    "@openzeppelin/contracts": "^2.5.0",
+    "@openzeppelin/contracts": "^3.0.0",
```

Comment:
Oh. I see you already saw that too! Are you a wizard?

Reply:
dependabot told me :wink: